project_type/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 ADHOC SA (http://www.adhoc.com.ar)
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Project Types",
+    "version": "18.0.1.0.2",
```

Comment:
```suggestion
    "version": "18.0.1.0.0",
```

Reply:
@Ricardo-MC , Thank you for suggestion.it's fixed